reco_utils/evaluation/python_evaluation.py

```diff
@@ -10,14 +10,25 @@
     explained_variance_score,
 )
 
-from reco_utils.common.constants import (
-    DEFAULT_USER_COL,
-    DEFAULT_ITEM_COL,
-    DEFAULT_RATING_COL,
-    PREDICTION_COL,
-    DEFAULT_K,
-    DEFAULT_THRESHOLD,
-)
+try:
```

Comment:
this try except is weird to me, in which case `from reco_utils.common.constants import ...` is going to raise an exception?

Reply:
I think I added this part awhile back when I wanted to upload scripts to AzureML workspace as few as possible... so I tried to remove any tiny module dependencies. But now just I'm uploading whole repo. Will remove this codes.